# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qt6webengine/hotfix-gn-deps.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/3rdparty/chromium/content/browser/BUILD.gn.orig        2023-02-27 18:57:08.000000000 +0000
+++ ./src/3rdparty/chromium/content/browser/BUILD.gn
@@ -36,6 +36,8 @@ jumbo_static_library("devtools_protocol"   deps = [
     "//base",
+    "//components/spellcheck:buildflags",
+    "//ui/gl:buildflags",
     "//content/browser/devtools:devtools_background_services_proto",
     "//content/browser/devtools:protocol_sources",
     "//content/common:buildflags",
--- qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/chrome/test/chromedriver/BUILD.gn.vanilla	2024-04-25 21:14:51.985000000 +0200
+++ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/chrome/test/chromedriver/BUILD.gn	2024-04-25 21:23:06.107000000 +0200
@@ -432,6 +432,7 @@
       ":automation_client_lib",
       ":lib",
       "//build:chromeos_buildflags",
+      "//ui/gl:buildflags",
       "//build/win:default_exe_manifest",
       "//mojo/core/embedder",
       "//net/server:http_server",
